config.json

```diff
@@ -14,5 +14,9 @@
   "call_add_cards_hook": false,
   "enable_debug_log": false,
   "search_the_web": false,
-  "timeout_seconds": 60
+  "timeout_seconds": 60,
+  "show_extended_filters": true,
```

Comment:
This config key show_extended_filters appears unused. I assume it needs to be implemented in a different PR.